packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/OverviewTabPanel.tsx

```diff
@@ -120,6 +187,28 @@ export const OverviewTabPanel = ({
         ]}
       />
 
+      {Boolean(
+        creditNoteQuery?.data || isCreditNoteLoading || creditNoteError
+      ) && (
+        <Box
+          sx={{
+            '& > * + *': {
+              mt: 2,
+            },
+          }}
+        >
+          <Typography variant="subtitle2" sx={{ mb: 2 }}>{t(
+            i18n
+          )`Linked documents`}</Typography>
+          {isCreditNoteLoading && <Skeleton variant="text" />}
+          <LinkedDocumentsCard
+            data={transformCreditNotes(
```

Comment:
* I would prefer to pass raw data inside the component.
* I would prefer to have a better naming for the `data` prop.

Reply:
Btw, what do you think about to move this `transformCreditNotes` into the component?